### DISCUSSION TITLE
es

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#13557] [Docs]
# es

Discover Ultralytics YOLOv8 - the latest in real-time object detection and image segmentation. Learn its features and maximize its potential in your projects.

https://docs.ultralytics.com/es

**Comment by uri-el99** [comment]
Hola, equipo de Ultralytics, primero que nada, quiero agradecer por su maravillo contribución con la octava versión de Yolo, es súper interesante y un modelo sumamente interesante.

Actualmente estoy utilizando Yolov8 para el desarrollo de mi proyecto de maestría. Hemos tenido resultados considerables, y nos hemos planteado publicar un artículo. 
Dicho lo anterior, me gustaría saber, ¿es necesaria una licencia, o algo similar para la publicación? 
Saludos cordiales.

**Reply by pderrenger** [reply]
@uri-el99 hola,

Gracias por tus amables palabras sobre YOLOv8 y por compartir tu experiencia positiva con el modelo. Nos alegra saber que está siendo útil para tu proyecto de maestría y que estás considerando publicar un artículo basado en tus resultados.

En cuanto a tu pregunta sobre la necesidad de una licencia para la publicación, aquí tienes la información relevante:

- **Uso Académico y Publicaciones**: Si estás utilizando YOLOv8 bajo la licencia AGPL-3.0 para fines académicos y planeas publicar tus resultados, no necesitas una licencia adicional. Sin embargo, es importante que cites adecuadamente el uso de YOLOv8 en tu trabajo y que cualquier código derivado que publiques también esté bajo la misma licencia AGPL-3.0.

- **Uso Comercial**: Si tu proyecto tiene fines comerciales o si planeas integrar YOLOv8 en productos o servicios comerciales, necesitarás una Licencia Empresarial de Ultralytics. Esta licencia permite el uso comercial sin los requisitos de código abierto de la AGPL-3.0.

Para más detalles sobre las licencias y para solicitar una Licencia Empresarial, puedes visitar [Ultralytics Licencias](https://ultralytics.com/license).

Si tienes alguna otra pregunta o necesitas más información, no dudes en preguntar. ¡Buena suerte con tu publicación y tu proyecto de maestría!

Saludos cordiales,
[Tu Nombre]
